AdvLoggerPkg/Library/AdvancedLoggerLib/MmCoreArm/AdvancedLoggerLib.c

```diff
@@ -87,7 +39,7 @@ ValidateInfoBlock (
   PcdAdvancedLoggerCarBase     -- NOT USED, leave at default
   PcdAdvancedLoggerPreMemPages -- NOT USED, leave at default
 
-  NOTE:  A debug statement here will cause recursion.  Insure that the recursion will be
+  NOTE:  A debug statement here will cause recursion. Ensure that the recursion will be
          a straight path just to return the existing mLoggerInfo.

```

Comment:
Looks like `mLoggerInfo` here should just be "logger info" now?

```suggestion
         a straight path just to return the existing logger information.
```

Reply:
Going to just update this since the reality here is bit different.